scrapy/spiderloader.py

```diff
@@ -23,6 +23,10 @@ def __init__(self, settings):
 
     def _load_spiders(self, module):
         for spcls in iter_spider_classes(module):
+            if spcls.name in self._spiders.keys():
+                import warnings
+                warnings.warn("There are several spiders with the same name (" + spcls.name +
+                              "), this can cause unexpected behavior", UserWarning)
```

Comment:
I think it'd be very helpful to point to modules with these spiders, at least to the latest module. Otherwise it can be hard to debug this issue.